src/main/java/coresearch/cvurl/io/request/ProxyAwareCVurlRequest.java

```diff
@@ -0,0 +1,132 @@
+package coresearch.cvurl.io.request;
+
+import coresearch.cvurl.io.mapper.BodyType;
+import coresearch.cvurl.io.model.CVurlConfig;
+import coresearch.cvurl.io.model.Response;
+
+import java.io.InputStream;
+import java.net.ProxySelector;
+import java.net.URI;
+import java.net.http.HttpResponse;
+import java.util.Optional;
+import java.util.concurrent.CompletableFuture;
+import java.util.function.Supplier;
+
+public class ProxyAwareCVurlRequest implements Request {
```

Comment:
The sole purpose of this wrapper was to remove proxies for current URI after request is done. It was removed, because now list of proxies is cleaned up inside the `CVurlProxySelector#select`